src/main/java/com/homework/mateakademy/GreetingController.java

```diff
@@ -0,0 +1,51 @@
+package com.homework.mateakademy;
+
+        import com.homework.mateakademy.domain.Message;
+        import com.homework.mateakademy.repo.MessageRepo;
+        import org.springframework.beans.factory.annotation.Autowired;
+        import org.springframework.stereotype.Controller;
+        import org.springframework.ui.Model;
+        import org.springframework.web.bind.annotation.GetMapping;
+        import org.springframework.web.bind.annotation.PostMapping;
+        import org.springframework.web.bind.annotation.RequestParam;
+
+        import java.util.List;
+        import java.util.Map;
+
+@Controller
+public class GreetingController {
+    @Autowired
+    private MessageRepo messageRepo;
```

Comment:
better use DI with constructor 
https://www.vojtechruzicka.com/field-dependency-injection-considered-harmful/

Reply:
in conntrolle use service layer
add service layer